src/flamegpu/simulation/CUDAEnsemble.cu

```diff
@@ -463,7 +812,7 @@ void CUDAEnsemble::setExitLog(const LoggingConfig &exitConfig) {
     // Set internal config
     exit_log_config = std::make_shared<LoggingConfig>(exitConfig);
 }
-const std::vector<RunLog> &CUDAEnsemble::getLogs() {
+const std::map<unsigned int, RunLog> &CUDAEnsemble::getLogs() {
```

Comment:
This will need to be in the changelog too. Probably worth diffing the include dir when this PR is good to merge to get the API changes.